app/controllers/posts_controller.rb

```diff
@@ -12,4 +12,8 @@ def show
     @comments_count = @post.comments.length
     @likes_count = @post.likes.length
   end
+
+  def new
+    @post = Post.new
+  end
```

Comment:
Great job so far @alphayowakarindi 

- [ ] Please bring back the `create post method` inside the `posts_controller.rb` file.
![backMethod](https://user-images.githubusercontent.com/55596970/202647191-a80f96c5-fc88-46b6-bc6a-5d0fc42813b3.png)

- [ ] I notice the `Method` was present but you accidentally remove it when you added the commit for `Add comments resource route`.
### **Please always test all your action method in your controllers before submitting for review**
**Happy Coding**

Reply:
Hi @CollinsTatang, thank you so much for taking the time to leave this excellent review. I've made the change. A lot of appreciation for your input to make my code better. Cheers!